app/assets/stylesheets/components/_tutorial.scss

```diff
@@ -13,3 +13,41 @@
     }
   }
 }
+
+.tutorial-header {
+  position: relative;
+}
+
+.show-all {
+  bottom: 0;
+  margin: 0;
+  padding: 0;
+  position: absolute;
+  right: 0;
+  width: auto;
+
+  img {
+    display: block;
+    width: 2rem;
+  }
+}
+
+.show-all,
+.show-all:hover {
+  background-color: transparent;
+}
+
+.single-page-tutorial {
+  .carousel-item {
+    margin-bottom: 6rem;
+  }
+
+  .owl-stage {
+    transform: translate3d(0px, 0px, 0px) !important;
```

Comment:
I haven't used translate3d myself; can you help me understand what a 0,0,0 translation is accomplishing here? What about the !important;?

Reply:
owl carousel uses this to scroll through the tutorial steps. if you scroll past the first item, this will now have some other value. i want to reposition everything back to the beginning when showing all, otherwise you see a blank screen.

!important is necessary because owl carousel applies transform in the html using the style attribute, rather than in a stylesheet. if i don't add !important it won't override that.